TypeCobol/Compiler/Sql/Scanner/DecimalFloatingPointLiteralTokenValue.cs

```diff
@@ -34,10 +34,15 @@ public class DecimalFloatingPointLiteralTokenValue : LiteralTokenValue
         public FloatingPointLiteralTokenValue NumericConstant { get; }
 
 
-        public DecimalFloatingPointLiteralTokenValue(LiteralTokenValueType type, FloatingPointLiteralTokenValue numericConstant) : base(type)
+        public DecimalFloatingPointLiteralTokenValue(FloatingPointLiteralTokenValue numericConstant) : base(LiteralTokenValueType.DecimalFloatingPoint)
         {
             NumericConstant = numericConstant;
         }
+        public DecimalFloatingPointLiteralTokenValue(bool isNegative, DecimalFloatingPointSpecialValueType decimalFloatingPointSpecialValue) : base(LiteralTokenValueType.DecimalFloatingPoint)
```

Comment:
The constructor should request the DecimalFloatingPointSpecialValue but not instanciate it.